web/docs/integrations/zscaler/README.md

```diff
@@ -0,0 +1,119 @@
+# Zscaler
+
+Zscaler's [Nanolog Streaming Service (NSS)][nss] is a family of products that
+enable Zscaler cloud communication with third-party security solution devices
+for exchanging event logs. You can either user Zscaler's Cloud NSS or deploy an
+on-prem NSS server to obtain the logs. Tenzir can receive Zscaler logs in either
+case.
+
+[nss]: https://help.zscaler.com/zia/understanding-nanolog-streaming-service
+
+![Zscaler NSS](zscaler-nss.svg)
+
+## Configure Cloud NSS to send events to Tenzir
+
+### Configure Tenzir
+
+First, spin up a Tenzir pipeline that mimics a [Splunk HEC
+endpoint](../splunk/README.md):
+
+```tql
+from_fluent_bit "splunk", options={
+  listen: 0.0.0.0,
+  port: 8088,
+  splunk_token: YOUR_TOKEN,
+}
+publish "zscaler"
+```
+
+In the above example, the pipeline uses `0.0.0.0` to listen on all IP addresses
+available.
+
+### Create a Cloud NSS Feed
+
+Perform the following steps to create a Cloud NSS feed:
+
+1. Log in to your ZIA Admin Portal and go to *Administration → Nanolog Streaming
+   Service*.
+2. Select the *Cloud NSS Feeds* tab.
+3. Click *Add Cloud NSS Feed*.
+
+In the new dialog, configure the following options:
+
+- **Feed Name**: Enter a name, e.g., `Tenzir ZIA Logs`
+- **NSS Type**: NSS for Web
+- **Status**: Enabled
+- **SIEM Rate**: Unlimited
+- **SIEM Type**: Other
+- **OAuth 2.0 Authentication**: disabled
+- **Max Batch Size**: 16 KB
+- **API URL**: Enter the URL that identifies the Tenzir pipeline where the
+  Splunk HEC endpoint is listening, e.g.,
+  https://1.2.3.4:8080/services/collector.
+- **HTTP Headers**: Add your token from the Tenzir pipeline enable compression
+  with the following two headers.
+  - `Authorization`: `YOUR_TOKEN`
+  - `Content-Encoding`: `gzip`
+- **Feed Output Type**: JSON
+- **JSON Array Notation**: disabled
+- **Feed Escape Character**: `,\"`
+
+After a web, firewall, or DNS feed has been configured, activate the changes as
+needed and test the feed. To test the connection:
+
+1. Go to *Administration* → *Nanolog Streaming Service* → *Cloud NSS Feeds*.
+2. Click the *Cloud icon*. This sends a test message to the Cribl receiver.
+
+## Configure an NSS Feed to send events to Tenzir
```

Comment:
I think this would be easier to navigate if you turn this into a separate integration. I stumbled upon this when first reading this, thinking that this was only about Cloud NSS.

Reply:
We discussed this in a call. Matthias feels strongly about not separating this into two pages, so I'll yield on this one.